pop3/commands/dele/dele.go

```diff
@@ -10,11 +14,34 @@ import (
 
 //TODO add a config variable to only mark as deleted
 func Process(c *Client, cmd Command) Reply {
-	res := OK
-	msg := ""
 	if c.State != Transaction {
-		res = ERR
-		msg = "invalid state"
+		return Reply{Result: ERR, Message: "invalid state"}
+	}
+	if cmd.Args == "" {
+		return Reply{Result: ERR, Message: "index required"}
+	} else if cmd.Args != "" {
+		envs := db.List(c.GetString("authenticated_as"), true)
+		deleted := c.GetStringSlice("deleted")
```

Comment:
its a non issue, it implicitly gets built by append. make([]T, 0) makes the linter upset as well (we have tons of linter issues)